### DISCUSSION TITLE
Change Default color of date time picker

### DISCUSSION THREAD

**Discussion by MariamIdeen** [mmazzarolo/react-native-modal-datetime-picker#677] [Q&A] [answered]
How to change the color of the date time picker according to the app theme. (Need to change blue to green)
![Screenshot](https://user-images.githubusercontent.com/117250792/206965268-696776b7-0200-4a75-ad2a-549c31728365.jpeg)

**Comment by KAHEL-HAN** [comment, marked as answer]
[How do I change the color of the Android date and time pickers?](https://github.com/mmazzarolo/react-native-modal-datetime-picker#how-do-i-change-the-color-of-the-android-date-and-time-pickers)

I found what you need at README.md
